### DISCUSSION TITLE
How to call Custom Report Sender class in ACRA 5.9.7

### DISCUSSION THREAD

**Discussion by DivyaBasatwar** [ACRA/acra#1205] [Q&A]
When android app gets crashed, then I want to give options to user where he would like send crash, like by sending mail, or via Bluetooth, on WhatsApp, etc. In order to do this, I will need to implement Report Sender Class where I can call implicit intent to send crash report with an action **Intent.ACTION_SEND** , so that user can select app to send his crash report. (Correct me if I am wrong or is there any other way to get this thing done). But, in **ACRA 5.9.7** I am not getting how to call that Custom Report Sender class in application class as in previous versions of ACRA this was getting done using annotations but, those annotations has been removed in the latest version of ACRA. Also, can we call implicit intent with action **Intent.ACTION_SEND** in this custom report sender class to send crash report.

**Comment by F43nd1r** [comment]
https://www.acra.ch/docs/Custom-Extensions

**Reply by DivyaBasatwar** [reply]
You have mentioned how to create custom extension, but, how to call that MyAdmin method in application. and can we get crash report in that custom extension.

**Reply by F43nd1r** [reply]
Sorry, it is completely unclear what you're asking. The docs cover how to configure and extend ACRA.
